lib/cisco_node_utils/ace.rb

```diff
@@ -0,0 +1,225 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
```

Comment:
I'd like to put this review on hold for now. The ACL code in #111 looks pretty good (once final comments are addressed) so I'd like to see that one get wrapped up and merged soon because I have some thoughts on this ace code that will rely on the ACL code being in place and will hopefully simplify the ace code a bit more. I think we should have a short design meeting tomorrow or monday to discuss.